y-board-v3/hardware_test/src/main.cpp

```diff
@@ -1,79 +1,251 @@
+#include <Adafruit_GFX.h>
+#include <Adafruit_NeoPixel.h>
+#include <Adafruit_SSD1306.h>
+#include <Arduino.h>
+#include <Wire.h>

```

Comment:
I think you can remove all of these (except for maybe Arduino.h>. They should already be included with yboard.h.